docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[ml-biases]]
+=== Possible biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. People use {ml} in science, healthcare, business, 
+entertainment, and many other fields. Decisions are made backed by {ml} 
+techniques. In such a setup, it is crucial to keep in mind the possible biases 
+that can corrupt results, endanger fairness, and jeopardize the success of even 
+the best-intended initiative. Some of these social biases can make their way to 
+the {ml} analytics results more easily than we might think.
+
+There are various possible biases that can negatively influence the result of a 
+{ml} analysis. *These biases typically are not present in the {ml} algorithms 
+that are used*, but rather exist in the datasets themselves or in the way the 
+results are interpreted. We take a look at the two main types of biases: data 
+related ones and interpretation related ones and recommend some basic practices 
+to avoid them. However, it is still important to remember that the problems 
+thematized in this piece are not solved yet. In fact, they are only a part of a 
+greater question about the ethics behind technology.
+
+
+[float]
+==== Biases in datasets
+
+Built-in biases are inserted in the data that we use to train the {ml} model. 
```

Comment:
Inserted in seems to imply an intentional activity. Could we change to "exist" instead?
```suggestion
Built-in biases exist in the data that we use to train {ml} models. 
```

Reply:
Sure! Great sentence, thank you!